homeassistant/components/tasmota/sensor.py

```diff
@@ -141,7 +152,10 @@
         STATE_CLASS: SensorStateClass.MEASUREMENT,
     },
     hc.SENSOR_PROXIMITY: {ICON: "mdi:ruler"},
+    hc.SENSOR_REACTIVE_ENERGYEXPORT: {DEVICE_CLASS: SensorDeviceClass.ENERGY},
+    hc.SENSOR_REACTIVE_ENERGYIMPORT: {DEVICE_CLASS: SensorDeviceClass.ENERGY},
```

Comment:
This is not correct, `SensorDeviceClass.ENERGY` should be `kWh`, reactive energy is in `kvarh` or similar.
Also, this should have a state class, I guess state class `TOTAL`, with or without last_reset: https://developers.home-assistant.io/docs/core/entity/sensor/#how-to-choose-state_class-and-last_reset